### ISSUE TITLE
0.5 is a Card

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-GUM#62]
It seems the UD standard is to label things like 0.5 a Card, not a Frac

**Comment by amir-zeldes** [comment]
This seems so counterintuitive to me that I would like to suggest this should be discussed again. What do you think @dan-zeman @nschneid ? 

I don't see why "one" and "1" should have the same NumType but "half" and "0.5" shouldn't.

**Comment by nschneid** [comment]
Because "0.5" can be read as "zero point five" or just "point five". It is only through mathematical knowledge (not linguistic knowledge) that we know this is equivalent to 1/2 = one half.

**Comment by nschneid** [comment]
I actually wouldn't object to adding `NumType=Dec` for numbers with a decimal point. But that would result in "5.0" and "5" being different NumTypes. That real numbers with only zeros after the decimal point are also integers is mathematical, not linguistic, knowledge.

**Comment by dan-zeman** [comment]
I don't have much to add here. I'd be fine with using just `NumType=Card`, wouldn't need `Frac`. `Frac` is a Czech-specific artefact of conversion from PDT, where it is used for words like _polovina_ "half", _třetina_ "third", _čtvrtina_ "fourth" etc. These inflect exactly like feminine nouns but are categorized under numerals. The feature value is not used for other numerals that represent non-integer numbers. If you say _tři čtvrtiny_ "three fourths", the first word will be regular `NUM NumType=Card`, the second word will be `NUM NumType=Frac` (http://hdl.handle.net/11346/PMLTQ-BSCI). If you say _3/4_ or _0.75_, it will be simply a cardinal number. There seems to be an inconsistency in _polovina_ "half" because I just found out that it's actually a `NOUN` (http://hdl.handle.net/11346/PMLTQ-RJ6K). And the other word for "half", _půl_, is `NUM NumType=Card` (or `NOUN`) (http://hdl.handle.net/11346/PMLTQ-BWJU); it does not behave morphologically like those _-ina_ words.

To sum up, I suppose that the original idea was that `NumType=Frac` is used to denote a subclass of cardinals which have specific morphosyntactic properties and which also happen to be one possible way of denoting fractions. And the idea was not applied consistently (because it should also be applied to _polovina_).

**Comment by nschneid** [comment]
In English I suppose there is value in distinguishing the two senses of "third", the fractional one and the ordinal one, both of which are separate from cardinal "three". Corresponding to cardinal "two" are the fractional "half" and ordinal "second".

A simple rule could be that NumType is strictly for the morphological paradigm, so in the case of words formed from digits without overt morphological marking, NumType=Card applies regardless of what is represented mathematically.
